2023-round-1/sirichandana-v/nx_tutorial_script.py

```diff
@@ -0,0 +1,24 @@
+import networkx as nx
+import matplotlib.pyplot as plt
+
+DG = nx.DiGraph()
+t = ('Hyd', 3)
+DG.add_nodes_from([1, 2, 'Delhi', t, 4])
+DG.add_edges_from([(1, 'Delhi'), ('Delhi', 1), (1, t), (4, t), (2, 4), (1, 2)])
+
+options = {
+    'width': 3,
+    'node_size': 1500
+}
+
+sp = dict(nx.all_pairs_shortest_path(DG))
+
+for key in sp:
+    for val in sp[key]:
+        l = len(sp[key][val])-1
+        if key != val:
+            print("Shortest Distance between {} and {} is {}".format(key, val, l))
+
```

Comment:
There is no need to find the path length, you just need to print out the path from one node to another.

Reply:
oh..I will change that and print the path. Thank you.

Reply:
@MridulS  I have updated the code. Could you pls check this and suggest me any other changes needed. I have also included #8 in this PR.